ocaml/xapi/cli_operations.ml

```diff
@@ -1871,7 +1885,9 @@ let select_vms ?(include_control_vms = false) ?(include_template_vms = false) rp
   let params = if not include_template_vms then ("is-a-template"    , "false") :: params else params in
   let vm_name_or_ref = try Some (
       (* Escape every quote character *)
-      List.assoc "vm" params |> String.replace "\"" "\\\""
+      let quote = Re.(compile (str {|"|})) in
```

Comment:
you could compile the regex just once if you put this somewhere global

Reply:
I've moved it outside the function and reused it for sr and host parameters.